chromium/background-scripts/rules.js

```diff
@@ -232,26 +241,33 @@ RuleSets.prototype = {
     this.addFromJson(rulesets, scope);
   },
 
+  /**
+   * @param {*} ruleJson
+   * @param {string} scope
+   */
   addFromJson: function(ruleJson, scope) {
-    const scope_obj = getScope(scope);
-
     if (this.wasm_rs) {
       this.wasm_rs.add_all_from_js_array(
         ruleJson,
         settings.enableMixedRulesets,
         this.ruleActiveStates,
-        scope);
+        getScope(scope));
```

Comment:
Why change this string to a scope in JavaScript?  The scope will be cached when passed to Rust at any rate.  It looks like this passes a RegExp to the Rust bindings, which it isn't expecting.

Reply:
Look like a mistake made when I was trying to figure out the types of `scope` in different calls. The original source code uses string `scope` too. It should be a string. Changes made in f87434f

As the `getScope` function is being called in `Ruleset` constructor exclusively, I added 1c20e82 to reflect this. Only the internal representation of the `scope` is a `RegExp` now. Thanks.